### PR TITLE
Fix negative `BigInt` formatting

### PR DESCRIPTION
**Pull request by ryangoree**
Closes: #4081

### CODE REVIEW

File: crates/js-sys/src/lib.rs [hunk 1]
```diff
@@ -1493,21 +1502,33 @@ impl fmt::Debug for BigInt {
 impl fmt::Display for BigInt {
     #[inline]
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.pad_integral(self >= &BigInt::from(0), "", &self.to_string_unchecked(10))
+        f.pad_integral(
+            self >= &BigInt::from(0),
+            "",
+            &self.abs().to_string_unchecked(10),
+        )
```
Comment (daxpedda) [inline crates/js-sys/src/lib.rs:1509]: The check `self >= &BigInt::from(0)` is being done twice now, probably the easiest thing here is to let `abs()` also return a `bool`.

**Review by daxpedda — CHANGES_REQUESTED** [review]
This needs an entry in the changelog.
Thank you!

**Comment by ryangoree** [comment]
> This needs an entry in the changelog. Thank you!

I added an entry to the top-level changelog which seemed to be the convention, but lmk otherwise.

File: crates/js-sys/src/lib.rs [hunk 2]
```diff
@@ -1388,12 +1388,12 @@ impl BigInt {
             .unchecked_into()
     }
 
-    /// Returns the absolute value of this BigInt value.
-    fn abs(&self) -> Self {
+    /// Returns a tuple of this BigInt's absolute value along with a boolean indicating whether the BigInt was negative.
```
Comment (daxpedda) [inline crates/js-sys/src/lib.rs:1391]:
```suggestion
    /// Returns a tuple of this [`BigInt`]'s absolute value along with a
    /// [`bool`] indicating whether the [`BigInt`] was negative.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -14,6 +14,9 @@
 * Fixed many proc-macro generated `impl` blocks missing `#[automatically_derived]`, affecting test coverage.
   [#4078](https://github.com/rustwasm/wasm-bindgen/pull/4078)
 
+* Fixed negative `BigInt` values being incorrectly formatted with 2 leading dashes.
```
Comment (daxpedda) [inline CHANGELOG.md:17]:
```suggestion
* Fixed negative `BigInt` values being incorrectly formatted with two minus signs.
```

**Review by daxpedda — APPROVED** [review]
Thank you!

**Pull request merged**